config/routes.rb

```diff
@@ -347,6 +347,7 @@
 
     # Chat channel
     patch "/chat_channels/update_channel/:id" => "chat_channels#update_channel"
+    post "/create_channel" => "chat_channels#create_channel"
```

Comment:
Just out of curiosity, the standard POST to `/chat_channels` isn't an option here?